container/test-app/iframe/microfrontend-sandbox-rules.html

```diff
@@ -0,0 +1,14 @@
+<!-- This microfrontend HTML file is used to test sandbox rules functionality -->
```

Comment:
I can see you have added this file  [microfrontend-sandbox-rules.html](https://github.com/SAP/luigi/pull/3844/files#diff-1bccb4094c8078d729a629df50f0d7a6f7f2bc1e2d9525c96ae05b6bbb55dbbd) and I assume that we might need to add more as needed in the future . 

It seems like we might be "running out of space" as shown in the picture below, as we keep adding more and more containers: 
<img width="966" alt="Screenshot 2024-08-07 at 15 42 56" src="https://github.com/user-attachments/assets/1260ebc3-391d-4182-ae7c-e91c0990952d">

To keep things separated to avoid the previous clutter that we had to restructure, I would expect to have another category for such types of functionality. 
<img width="687" alt="Screenshot 2024-08-07 at 15 44 11" src="https://github.com/user-attachments/assets/7cfde02c-2a8b-4535-b8dd-5d35bdbddc80">

So for example we can add an <Add new Category Here> where we put the sandbox rules test, defer init test, and maybe soon also the allowRules test. I don't know what we can call this category, but it would be nice to start separating now, then keep restructuring in the future.